canvas_course_info/settings/base.py

```diff
@@ -140,6 +140,8 @@
 ICOMMONS_API_TOKEN = SECURE_SETTINGS.get('icommons_api_token')
 ICOMMONS_BASE_URL = SECURE_SETTINGS.get('icommons_base_url')
 ICOMMONS_API_PATH = '/api/course/v2/'
+ICOMMONS_REST_API_SKIP_CERT_VERIFICATION = True
```

Comment:
Good catch, I started off as True to quickly verify it works , but meant to modify it to False along the lines of the other tool. Will fix it